blazor/treeview/authorization-and-authentication.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: Authorization,Authentication in Blazor TreeView Component | Syncfusion
+description: Checkout and learn here all about Authorization and Authentication in Syncfusion Blazor TreeView component and much more details.
+platform: Blazor
+control: TreeView
+documentation: ug
+---
+
+# Authorization and Authentication in Blazor TreeView Component
+
+**Authentication** is the process of verifying the identity of a user or system. This is typically done by requiring a username and password, but can also include other forms of verification such as biometric data or security tokens.
+
+**Authorization**, on the other hand, is the process of determining whether a user or system has access to a particular resource or action. Once a user's identity has been authenticated, the system can then determine whether they are authorized to perform a specific action or access a specific resource. This is often done by comparing the user's credentials or permissions against a set of rules or policies.
+
+Provides a sample of Authorization and Authentication that explains how authorized users can access the TreeView. The below blog also includes steps to create a [Blazor Server App with Authentication](https://www.syncfusion.com/blogs/post/easy-steps-create-a-blazor-server-app-with-authentication.aspx), ensuring easy setup and configuration.
+
+The Blazor TreeView component will be shown for authorized users, and a message to either log in or register will be displayed as specified in the not authorized users.

```

Comment:
In the above example, the Blazor Server App is equipped with authentication, which ensures that only authorized users can access TreeView. @PrasanthMadhaiyan - Can the content be modified like this?